pyproject.toml

```
@@ -1,12 +1,12 @@
-[build-system]
-requires = ["setuptools>=61.0"]
-build-backend = "setuptools.build_meta"
-
-[project]
-authors = [
-    {name = "Saritasa", email = "pypi@saritasa.com"},
-]
-requires-python = ">=3.9"
+[tool.poetry]
+name = "django-import-export-extensions"
+version = "0.4.2"
```

Current version is `0.4.1`, the next one will be `0.5.0` probably (or 1.0.0)

```suggestion
version = "0.4.1"
```